apps/react/src/features/genres/components/GenreCard/GenreCard.tsx

```diff
@@ -11,16 +12,18 @@ type Props = {
 	readonly genre: Genre;
 };
 
-const GenreCardComponent: FC<Props> = ({ genre }: {genre: Genre;}) => (
-	<ListItem disablePadding>
-		<ListItemButton className={styles['genre-card']}>
+const GenreCardComponent = forwardRef<HTMLLIElement, Props>(({ genre }: Props, ref) => {
+	const location = useLocation();
+
+	return <ListItem ref={ref} disablePadding>
+		<ListItemButton className={styles['genre-card']} component={Link} to={`${genre.id}${location.search}`}>
```

Comment:
Let's add a variable with a link: 
```suggestion
	const location = useLocation();
	const genreCardLink = `${genre.id}${location.search}`

	return <ListItem ref={ref} disablePadding>
		<ListItemButton className={styles['genre-card']} component={Link} to={genreCardLink}>
```